src/client/datascience/types.ts

```diff
@@ -148,7 +148,7 @@ export interface INotebook {
 export type ConnectNotebookProviderOptions = {
     getOnly?: boolean;
     ui: IDisplayOptions;
-    localOnly?: boolean;
+    localOnly: boolean;
```

Comment:
Planning on renaming this, basically to mean `start local server` (i.e. when we start jupyter for local purposes).
I thought that's what this mean, but that's not true.

Reply:
Will need a good meaningful name